onnx/test/reference_evaluator_model_test.py

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) ONNX Project Contributors
+
+# SPDX-License-Identifier: Apache-2.0
+# type: ignore
+import unittest
+
+import numpy as np
+
+import onnx
+import onnx.helper as oh
```

Comment:
I would avoid aliasing the modules since these aliases are unconventional. Maybe something like

from onnx import helper, numpy_helper

will suffice?

for reference I would use the fully qualified name since it is not really referenced in many places

Reply:
The instructions become very long and they get split over multiple lines. It is difficult to get the whole function without scrolling.